content/posts/newsletter-november-2024.md

```diff
@@ -0,0 +1,36 @@
+title: Read the Docs Newsletter - November 2024
+date: November 11, 2024
+description: Company updates and new features from the last month, current focus, and upcoming features.
+category: Newsletter
+tags: newsletter, community
+authors: Eric Holscher
+status: published
+image: /images/november-2024.jpg
+image_credit: Photo by <a href="https://unsplash.com/@hermanndoyo?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Doyo Hermann</a> on <a href="https://unsplash.com/photos/a-scenic-view-of-a-town-surrounded-by-mountains-y1VQvwSqYOE?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a>
+
+
+## News and updates
+
+* The new **Community dashboard** at https://app.readthedocs.org will be made the default dashboard on November 12. We are still beta testing the Business dashboard, as it effects documentation authentication and requires a bit more testing. We expect to make the Business dashboard the default soon.
+* We've added **Multi-Factor Authentication (MFA)** to our platform, and it's now fully released. Give it a try to add an extra layer of security to your account.
+* We have added **Python 3.13** as a build option, along with `updating other build tools <https://github.com/readthedocs/readthedocs.org/pull/11742>`_. 
+* **Addons** continues to get iterative improvements, including allowing more granular configuration of documentation page documentations, and fixing bugs in the injection logic for large images.
```

Comment:
What are we referring to with `documentation page documentations`?

Reply:
Ha, notifications 🙃 
```suggestion
* **Addons** continues to get iterative improvements, including allowing more granular configuration of `documentation notifications <https://docs.readthedocs.io/en/stable/doc-notifications.html>`_, and fixing bugs in the injection logic for large images.
```